src/domain/dto/readInstallableServicesItems.go

```diff
@@ -0,0 +1,18 @@
+package dto
+
+import (
+	"github.com/goinfinite/os/src/domain/entity"
+	"github.com/goinfinite/os/src/domain/valueObject"
+)
+
+type ReadInstallableServicesItemsRequest struct {
+	Pagination Pagination                 `json:"pagination"`
+	Name       *valueObject.ServiceName   `json:"name,omitempty"`
+	Nature     *valueObject.ServiceNature `json:"nature,omitempty"`
+	Type       *valueObject.ServiceType   `json:"type,omitempty"`
```

Comment:
The Request/Response DTOs usually carry not only characteristics regarding the type of item being fetch, but also users filters and may also correlate different types of items. If you look at how it was done on Ez, you'll see that I made sure to use "Container" as prefix there:
https://github.com/goinfinite/ez/blob/main/src/domain/dto/readContainers.go#L10

That way I can make sure I'm referring to the Container and not to something else.

The same applies for the Response DTO. "Items" is way too generic. On Ez you can see that I have two types of returns, Containers and ContainersWithMetrics. See? The necessity exists. Request/Response DTOs aren't limit to a particular entity, it's much wider context and therefore these small details such as prefixes matter.

I would let "Items" pass this time on the Response, but I think you should start following this practice then using Request/Response DTOs. You'll also need to adjust the front-end code again I suppose.